apps/web/next.config.mjs

```diff
@@ -49,6 +49,9 @@ const nextConfig = withPWA({
       },
     ];
   },
+  env: {
+    MEILISEARCH_ENABLED: !!process.env.MEILI_ADDR,
+  },
```

Comment:
This is a build time env variable so not exactly what we want. Also next now advices against using this in their docs (https://nextjs.org/docs/pages/api-reference/next-config-js/env). Let's use our custom config mechanism in `shared/config.ts` instead for consistency.